Makefile

```diff
@@ -34,3 +34,11 @@ publish_release_testpypi: build_release
 
 publish_release: build_release
 	twine upload dist/*
+
+.dev:
+	pip install -e .[test,docs,devpy36]
+	touch .dev
+
+lint: .dev
+	pylint setup.py oscarapi/
+	black -l99 --check --exclude "migrations/*" setup.py oscarapi/**/*.py
```

Comment:
Please do not change the black defaults, it is black or nothing.

Reply:
I changed it to the black defaults